OracleTuxedo/salt/samples/bankapp/README.md

```diff
@@ -0,0 +1,54 @@
+
+# Oracle Tuxedo `bankapp` Sample Container Image
+
+This sample extends the Oracle Tuxedo image by creating a "bankapp" application.
+
+## Prerequisites
+
+This image uses the Orace Tuxedo container image `oracle/tuxedo:latest` as its base. Please follow the [Oracle Tuxedo image](https://github.com/oracle/docker-images/tree/main/OracleTuxedo/core) documentation to build the container image before performing the operations described in this document.
+
+## How to Build the Image
+
+Download the parent directory (bankapp) of this README.md file from GitHub to a location of your choice. This location is referenced as `YOUR_APP_DIR` in this document.
```

Comment:
```suggestion
Download the parent directory (`bankapp`) of this `README.md` file from GitHub to a location of your choice. This location is referenced as `YOUR_APP_DIR` in this document.
```

Reply:
Please use `$APP_DIR` instead as it's shorter. It should be defined so that your commands could work too.

Reply:
Done.